lib/virt_autotest/utils.pm

```diff
@@ -169,8 +169,14 @@ sub is_kvm_host {
 #retrun 1 if libvirt 9.0- is running which monolithic libvirtd is the default service
 sub is_monolithic_libvirtd {
     record_info('WARNING', 'Libvirt package is not installed', result => 'fail') if (script_run('rpm -q libvirt-libs'));
-    unless (is_alp) {
-        return 1 if script_run('systemctl is-enabled libvirtd.service') == 0;
+    if (is_alp) {
+       return 0;
+    }
+    if (is_tumbleweed) {
+       return 1 if script_run('rpm -q libvirt-libs | grep -e "libs-9\.0" -e "libs-[1-8]\."') == 0;
```

Comment:
It would be better do not use libs version detecting. Please refer to [here](https://github.com/SUSE/qa-automation/blob/d2d4d5c8bf2c79b6aa2a416cb8b126938d0d9f05/libs/qa_lib_virtauto/qa_lib_virtauto/lib/virtlib#L2232C21-L2232C21)